dev/fixGrammarTypes.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs-extra');
```

Comment:
Can you add a comment at the top of the file indicating why we need this script and what it does?